packages/health/android/build.gradle

```diff
@@ -46,6 +46,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation("com.google.android.gms:play-services-fitness:21.1.0")
     implementation("com.google.android.gms:play-services-auth:20.2.0")
-    // The new health connect api
+    // TODO: Checking this SDK & SDK in health package for Android 14
+    // https://developer.android.com/jetpack/androidx/releases/health-connect#1.1.0-alpha03
```

Comment:
we dont need this change here please remove.

Reply:
From Android 14 we need to be upgrade version of Health Connect SDK, It is a prerequisite for Android 14. Apps will not be able to integrate with Health Connect on Android 14 without it -> So I added TODO.
Please check it here: https://developer.android.com/jetpack/androidx/releases/health-connect#1.1.0-alpha01

Reply:
will you use android sdk from 14 not the flutter SDK?
i think flutter SDK health package will handle it so we just need to check flutter health package?

Reply:
@hibitness-dinakar-maurya For normal cases, installing plugins following the official way and using plugins developed by a large team, waiting for updates from the development team usually doesn't take too much time. However, for instance, in the case of Android 14 which has been out for a while now but the plugin hasn't been updated and a new release version hasn't been made available, we are compelled to continue maintaining this forked custom branch to add additional plugins to support Android 14. Therefore, this comment simply serves as a note that if in the future we still need to use this forked branch

Reply:
delete above message from gradle file